source/render.py

```diff
@@ -30,22 +30,6 @@ def generateHTML(prefix):
         panelists = "".join([x if i == 0 else ", "+x for i,x in enumerate(row[5:9]) if x != ""])
         day1_32.append([time]+row[2:5]+[panelists])
 
-    day2_26= []
```

Comment:
if it ain't broke don't fix it, we will need this code in about a month, it's easier if names can just be put into the spreadsheet and this code just works to generate an html file